exercises/nth-prime/README.md

```diff
@@ -0,0 +1,5 @@
+Given a number n, determine what the nth prime is.

```

Comment:
The readme needs to be generated from the templates with the configlet. You can find more information in the `CONTRIBUTING.md` and in [Generating a README](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md#generating-a-readme)